lint/build.gradle

```diff
@@ -0,0 +1,40 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
+    }
+}
+
+apply plugin: 'kotlin'
+apply plugin: 'jacoco'
+
+targetCompatibility = JavaVersion.VERSION_1_7
+sourceCompatibility = JavaVersion.VERSION_1_7
+
+configurations {
+    lintChecks
+}
+
+jar {
+    manifest {
+        attributes('Lint-Registry': 'net.grandcentrix.thirtyinch.IssueRegistry')
+    }
+}
+
+repositories {
+    maven { url "https://dl.bintray.com/android/android-tools" }
```

Comment:
Yeah, I agree that it's unfortunate that we have to include the additional repository, just so we can access Lint-specific dependencies. A comment and, potentially, a link to an issue over at b.android.com would be helpful here!